windows_bat/PokemonGo-Bot-Install.bat

```diff
@@ -174,89 +216,210 @@ goto:Menu
 
 
 :getit
+ECHO.----- Checking Program Downloading ----->>%log%
+ECHO.>>%log%
+ECHO.Downloading "%~1%~2" to "%DownPath%\%~2">>%log%
+ECHO.>>%log%
 start /wait powershell -Command "Invoke-WebRequest %~1%~2 -OutFile %DownPath%\%~2"
 goto:eof
 
 
 :installit
+ECHO.----- Checking Program Installing ----->>%log%
+ECHO.>>%log%
+ECHO.Installing "%DownPath%\%~1 %~2">>%log%
+ECHO.>>%log%
 start /wait %DownPath%\%~1 %~2
 goto:eof
 
 
 
 :InstallBot
-CLS
-@ECHO --------------------Creating Backup--------------------
-@ECHO.
-@ECHO.
-@ECHO.
-if exist %PGBotPath%\configs\config.json copy %PGBotPath%\configs\config.json %DownPath%
-if exist %PGBotPath%\web\config\userdata.js copy %PGBotPath%\web\config\userdata.js %DownPath%
-if exist %PGBotPath%\encrypt. copy %PGBotPath%\encrypt. %DownPath%
+ECHO.----- Checking Choice and Bot Installing ----->>%log%
+ECHO.>>%log%
+ECHO.Choice was "%_ok%" installing Bot>>%log%
+ECHO.>>%log%
+cls
+@ECHO.--------------------Creating Backup--------------------
+@ECHO.
+@ECHO.
+@ECHO.
+ECHO.----- Checking Creating Backup ----->>%log%
+ECHO.>>%log%
+IF EXIST %PGBotPath%\encrypt.so (
+ECHO.Copying "%PGBotPath%\encrypt.so" to %DownPath%>>%log%
+COPY %PGBotPath%\encrypt.so %DownPath%>>%log%
+) else (
+ECHO.No "%PGBotPath%\encrypt.so">>%log%
+)
+IF EXIST %PGBotPath%\encrypt.dll (
+ECHO.Copying %PGBotPath%\encrypt.dll to %DownPath%>>%log%
+COPY %PGBotPath%\encrypt.dll %DownPath%>>%log%
+) else (
+ECHO.No "%PGBotPath%\encrypt.dll">>%log%
+)
+IF EXIST %PGBotPath%\encrypt_64.dll (
+ECHO.Copying "%PGBotPath%\encrypt_64.dll" to %DownPath%>>%log%
+COPY %PGBotPath%\encrypt_64.dll %DownPath%>>%log%
+) else (
+ECHO.No "%PGBotPath%\encrypt_64.dll">>%log%
+)
+IF EXIST %PGBotPath%\configs\config.json (
+ECHO.Copying "%PGBotPath%\configs\config.json" to %DownPath%>>%log%
+COPY %PGBotPath%\configs\config.json %DownPath%>>%log%
+) else (
+ECHO.No "%PGBotPath%\configs\config.json">>%log%
+)
+IF EXIST %PGBotPath%\web\config\userdata.js (
+ECHO.Copying "%PGBotPath%\web\config\userdata.js" to %DownPath%>>%log%
+COPY %PGBotPath%\web\config\userdata.js %DownPath%>>%log%
+) else (
+ECHO.No "%PGBotPath%\web\config\userdata.js">>%log%
+)
+ECHO.>>%log%
+cls
 @ECHO.
 @ECHO.
 @ECHO.
-@ECHO --------------------Downloading PokemonGo-Bot--------------------
+@ECHO.--------------------Downloading PokemonGo-Bot--------------------
 @ECHO.
 @ECHO.
 @ECHO.
-if exist %PGBotPath% rmdir %PGBotPath% /s /q
-if not exist %PGBotPath% md %PGBotPath%
+ECHO. Be patience... We are working hard to download and install the PokemonGo-Bot.
+ECHO.----- Checking Removing Bot Folder ----->>%log%
+ECHO.>>%log%
+IF EXIST %PGBotPath% rmdir %PGBotPath% /s /q
+IF EXIST %PGBotPath% (
+ECHO.Problem %PGBotPath% still exists

```

Comment:
Both above are smoothing up the text , the first one is what ppl see during installation, the second one goes into the log file